website/server/libs/bannedWords.js

```diff
@@ -0,0 +1,3 @@
+let bannedWords = ['TEST_PLACEHOLDER_SWEAR_WORD_HERE'];
+
+module.exports = bannedWords;
```

Comment:
This file should have a comment at the top so that programmers can choose to protect themselves from seeing the words. For example:

// CONTENT WARNING:
// This file contains slurs, swear words, religious oaths, and words related to addictive substance and adult topics.
// Do not read this file if you do not want to be exposed to those words. 
// The words are stored in an array called \`bannedWords\` which is then exported with \`module.exports = bannedWords;\`
// This file does not contain any other code.

After that there should be several blank lines or lines containing just comment markers. 

The banned word array should then start with the milder words and build up to the worst ones, but we can do that part later in the direct commit that puts the words into this file.